File: app/src/main/java/com/alvinhkh/buseta/kmb/model/network/KmbRouteBoundRes.java
package com.alvinhkh.buseta.kmb.model.network;


import com.alvinhkh.buseta.kmb.model.KmbRouteBound;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class KmbRouteBoundRes {

    @SerializedName("data")
    public ArrayList<KmbRouteBound> data;

    @SerializedName("result")
    public Boolean result;

    public String toString() {
        return "KmbSpecialRouteRes{data=" + this.data + ", result=" + this.result + '}';
    }

}

File: app/src/main/java/com/alvinhkh/buseta/kmb/model/network/KmbRouteBoundRes.kt
package com.alvinhkh.buseta.kmb.model.network


import com.alvinhkh.buseta.kmb.model.KmbRouteBound
import com.google.gson.annotations.SerializedName

import java.util.ArrayList

data class KmbRouteBoundRes(
    @SerializedName("data")
    var data: ArrayList<KmbRouteBound>? = null,
    @SerializedName("exception")
    var exception: String? = null,
    @SerializedName("result")
    var result: Boolean? = null
)
